From 4d598814d7740c7720d13266b64ed5be42f0f646 Mon Sep 17 00:00:00 2001
From: Stefan Berger <s.berger81@gmail.com>
Date: Sat, 12 Jan 2019 18:08:55 +0100
Subject: [PATCH 5/6] vendor: Add new playstore permission

New white-listed permission is needed for com.android.vending package.

Change-Id: I61a82610c4efb6d8a7f6066fb445bc2a55668bc8
---
 prebuilt/etc/permissions/privapp-permissions-elgoog.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/prebuilt/etc/permissions/privapp-permissions-elgoog.xml b/prebuilt/etc/permissions/privapp-permissions-elgoog.xml
index 599ef726..ec036fdb 100644
--- a/prebuilt/etc/permissions/privapp-permissions-elgoog.xml
+++ b/prebuilt/etc/permissions/privapp-permissions-elgoog.xml
@@ -84,6 +84,7 @@ It allows additional grants on top of privapp-permissions-platform.xml
         <permission name="android.permission.SEND_SMS_NO_CONFIRMATION"/>
         <permission name="android.permission.SET_PREFERRED_APPLICATIONS"/>
         <permission name="android.permission.STATUS_BAR"/>
+        <permission name="android.permission.SUBSTITUTE_NOTIFICATION_APP_NAME"/>
         <permission name="android.permission.UPDATE_DEVICE_STATS"/>
         <permission name="android.permission.WRITE_SECURE_SETTINGS"/>
     </privapp-permissions>
-- 
2.17.1

